test/metabase/api/user_key_value_test.clj

```diff
@@ -0,0 +1,12 @@
+(ns metabase.api.user-key-value-test
+  (:require [clojure.test :refer :all]
+            [metabase.test :as mt]
+            [toucan2.core :as t2]))
+
+(deftest it-works
+  (t2/delete! :model/UserKeyValue :user_id (mt/user->id :rasta))
```

Comment:
I think it's better to isolate this test with `(mt/with-model-cleanup)`, it'll make sure new keys are deleted after

Reply:
Ahh, good call, much better. Thanks!